vpr/src/route/rr_graph.cpp

```diff
@@ -741,11 +744,25 @@ void create_rr_graph(const t_graph_type graph_type,
     }
 
     if (is_flat) {
+        short delayless_switch = OPEN;
```

Comment:
Could make this a helper function: get_delayless_sw_index ().
Then add a Doxygen comment (right now the block off code is uncommented).